src/router.cpp

```diff
@@ -1,7 +1,7 @@
 #include "router.h"
 #include <esp_timer.h>
 
-#define GW_ANNOUNCEMENT_DEADLINE 5 * 10e6  // 5 seconds in microseconds
+#define GW_ANNOUNCEMENT_DEADLINE 6 * 10e6  // 1 minute in microseconds
```

Comment:
6.000.000 microseconds is 6 seconds not 60
```suggestion
#define GW_ANNOUNCEMENT_DEADLINE 60 * 10e6  // 1 minute in microseconds
```

Reply:
I double-checked on Google and Wolfram Alfa and it is correct 👯 

![image](https://user-images.githubusercontent.com/1025234/113597852-afec2a80-9612-11eb-80ad-0e10e18e2b91.png)

Reply:
I would test this delay on the actual code to check if its running on mili seconds or micro seconds and be sure that it will last what we are expecting.

Reply:
It is what we are expecting... The gateway announces itself every 10 seconds and the gateway address is valid for 1 minute.